src/components/MapView/Layers/ImpactLayer/index.tsx

```diff
@@ -24,18 +24,17 @@ import {
   mapSelector,
 } from '../../../../context/mapStateSlice/selectors';
 import { getFeatureInfoPropsData, getRoundedData } from '../../utils';
-import { safeTranslate } from '../../../../i18n';
+import { i18nTranslator, safeTranslate } from '../../../../i18n';
 
 const linePaint: LinePaint = {
   'line-color': 'grey',
   'line-width': 1,
   'line-opacity': 0.3,
 };
 
-function getHazardData(evt: any, operation: string) {
+function getHazardData(t: i18nTranslator, evt: any, operation: string) {
```

Comment:
Nice work. For all these functions, let's make `t` optional and move it as the last argument.